lib/bot.rb

```diff
@@ -99,4 +100,13 @@ def create_issue_and_update_github_pr_title
     Github::PullRequest.update_title(@repo, @pr_number, prefixed_title)
     new_issue
   end
+
+  def fix_pr_title
+    id = @title.match BRANCH_NAME_TICKET_ID_REGEX
+    return unless id
+
+    humanized_title = @title.gsub(id[0], "").tr("-", " ").strip.capitalize
```

Comment:
Instead of trying to change the title by changing the current title on Github, could we use the branch_name to create the new title?

Reply:
Looking at https://github.com/liefery/liefery-backend/pull/5631/commits It would be nice if we actually got the tittle from Jira and replaces the PR title then it doesn't exactly match the one from Jira, this would fix the title when it has some extra space at the beginning or when it is cut off by GitHub line length limit.

Reply:
That's a good point :) Didn't think about github line length